values.yaml

```diff
@@ -42,30 +46,22 @@ placeholder:
   maxSurge: 1
   maxUnavailable: 0
   minReadySeconds: 5
-  storeURL: "https://s3.amazonaws.com/pelias-data.nextzen.org/portland-metro/placeholder/store.sqlite3.gz"
+  storeURL: "https://data.geocode.earth/placeholder/2020-05-08/store.sqlite3.gz"
```

Comment:
original should be kept, this value should be added in an external values file to override this one. the external values file will not be added to this repo